common/personalization.js

```diff
@@ -134,9 +137,18 @@ function getPersonalizationOffer(aepEdgeResult, decisionScopeName) {
   );
 }
 
+function getPersonalizedDecisions(aepEdgeResult, surfaceUrl) {
```

Comment:
Should probably be getPersonalizationDecisions, for consistency with getPersonalizationOffer.